Assets/Scripts/Photos/PhotoAlbum.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+
+class PhotoAlbum : MonoBehaviour
+{
+    public RawImage albumPhoto, albumPhoto1, albumPhoto2, albumPhoto3;
+    public Button exitAlbumButton, nextPageButton, backPageButton;
+    public PhotoCamera photoCamera;
+    private int currentPage;
+    private int pageSize;
+
+    public void Start()
+    {
+        currentPage = 0;
+        pageSize = 4;
+
+        exitAlbumButton.onClick.AddListener(ExitAlbumOnClick);
+        nextPageButton.onClick.AddListener(NextPageOnClick);
+        backPageButton.onClick.AddListener(BackPageOnClick);
+        backPageButton.gameObject.SetActive(false);
+
+        LoadAlbumPage();
+    }
+
+    private void LoadAlbumPage(int skip = 0)
+    {
+        List<PhotoDTO> photos = PhotoCollectionDTO.PagePhotoCollection(pageSize, skip).ToList();
+
+        if(photos.ElementAt(0) != null)
+        {
+            Texture2D tex = new Texture2D((int)albumPhoto.rectTransform.rect.width, (int)albumPhoto.rectTransform.rect.height);
+            tex.LoadImage(photos.ElementAt(0).PhotoData);
+            albumPhoto.texture = tex;
+        }
+
+        if(photos.ElementAt(1) != null)
+        {
+            Texture2D tex1 = new Texture2D((int)albumPhoto1.rectTransform.rect.width, (int)albumPhoto1.rectTransform.rect.height);
+            tex1.LoadImage(photos.ElementAt(1).PhotoData);
+            albumPhoto1.texture = tex1;
+        }
+
+        if (photos.ElementAt(2) != null)
+        {
+            Texture2D tex2 = new Texture2D((int)albumPhoto2.rectTransform.rect.width, (int)albumPhoto2.rectTransform.rect.height);
+            tex2.LoadImage(photos.ElementAt(2).PhotoData);
+            albumPhoto2.texture = tex2;
+        }
+
+        if (photos.ElementAt(3) != null)
+        {
+            Texture2D tex3 = new Texture2D((int)albumPhoto3.rectTransform.rect.width, (int)albumPhoto3.rectTransform.rect.height);
+            tex3.LoadImage(photos.ElementAt(3).PhotoData);
+            albumPhoto3.texture = tex3;
+        }
+    }
+
+    private void ExitAlbumOnClick()
```

Comment:
the button functions aren't setting the camera state completely correctly atm. the projection plane is getting deactivated on discard / close album but the camera is still in first person mode. would recommend to synchronize s.t. projection plane is being de activated iff camera is being set to topdown mode.

Reply:
I think this is fixed